styles.css

```diff
@@ -0,0 +1,81 @@
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700&family=Poppins&display=swap');
+
+* {
+  box-sizing: border-box;
+  max-width: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-around;
+  padding: 24px;
+  margin-top: 58px;
+}
+
+.nav-bar a {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  text-decoration: none;
+}
+
+.fa-bars {
+  background: #6070ff;
+}
+
+#background-image {
+  background-image: url("assets/bg.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  max-width: 100%;
+  height: 250px;
+}
+
+.content {
+  padding: 24px;
+  margin-top: 0;
+}
+
+.content h1 {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  position: absolute;
+  top: 220px;
+}
+
+.connect {
+  padding: 24px;
+  position: absolute;
+  bottom: 15px;
+}
```

Comment:
- [ ] Your `Let's Connect` text seems to change positions when I alter the screen size. It might be because you have `position: absolute` on it so it varies itself according to the bottom of the body/ screen size. Kindly fix this issue so the text appear in the correct position as specified in the Figma template.
> Your Design
![image](https://user-images.githubusercontent.com/90556221/170392372-c675d02d-2b64-4587-a2c1-9533ecee021e.png)

- [ ] Please ensure all of your text have the correct `font-family`. This text should have a family of **Poppins** as specified in the Figma Design